frontend/integration-tests/tests/kubevirt/clone.vm.scenario.ts

```diff
@@ -0,0 +1,230 @@
+    },VM_BOOTUP_TIMEOUT);
+
+    it('Clone VM with URL source.', async() => {
+      await urlvm.action('Start');
+      await urlvm.action('Clone');
+      await wizard.next();
+
+      const clonedVm = new VirtualMachine(`${urlvm.name}-clone`, urlvm.namespace);
+      leakedResources.add(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+      await clonedVm.action('Start');
+
+      // Verify cloned PVC exists
+      await browser.get(`${appHost}/k8s/ns/${testName}/persistentvolumeclaims`);
+      await isLoaded();
+      await filterForName(`${clonedVm.name}-${urlvm.name}-rootdisk-clone`);
+      await resourceRowsPresent();
+
+      await clonedVm.action('Delete');
+      leakedResources.delete(JSON.stringify({name: clonedVm.name, namespace: clonedVm.namespace, kind: 'vm'}));
+    }, VM_BOOTUP_TIMEOUT * 2);
+
+    it('Clone VM with URL source and Cloud Init.', async() => {
+      await browser.get(`${appHost}/k8s/ns/${testName}/virtualmachines`);
+      await isLoaded();
+      await wizard.openWizard();
+
+      // Basic Settings
+      await wizard.fillName(cloudInitVmName);
+      await wizard.selectProvisionSource(cloudInitVmProvisionConfig);
+      await wizard.selectFlavor(basicVmConfig.flavor);
+      await wizard.selectOperatingSystem(basicVmConfig.operatingSystem);
+      await wizard.selectWorkloadProfile(basicVmConfig.workloadProfile);
+      await wizard.useCloudInit(cloudInitCustomScriptConfig);
+      await wizard.next();
+
+      // Networking
+      await wizard.next();
+
+      // Storage
+      await wizard.editDisk(1, 'size', '1');
+      await wizard.addDisk(hddDisk.name, hddDisk.size, hddDisk.StorageClass);
+
+      // Create VM
+      await wizard.next();
+      await wizard.waitForCreation();
+      leakedResources.add(JSON.stringify({name: cloudInitVmName, namespace: testName, kind: 'vm'}));
+
+      // Close wizard
+      await wizard.next();
+
+      // Clone VM
+      const ciVm = new VirtualMachine(cloudInitVmName, testName);
+      await ciVm.action('Clone');
+      await wizard.next();
+
+      const clonedVm = new VirtualMachine(`${cloudInitVmName}-clone`, testName);
+      leakedResources.add(JSON.stringify({name: clonedVm.name, namespace: testName, kind: 'vm'}));
+
+      // Check disks on cloned VM
+      const disks = await clonedVm.getAttachedResources(disksTab);
+
+      [hddDisk.name, 'cloudinitdisk'].forEach(element => {
+        expect(disks.includes(element)).toBe(true, `Disk ${element} should be present on cloned VM.`);
+      });
+
+      // Verify the cloned VM can boot
+      await clonedVm.action('Start');
+
+      // Delete VMs
+      await clonedVm.action('Delete');
+      leakedResources.delete(JSON.stringify({name: clonedVm.name, namespace: testName, kind: 'vm'}));
+      await ciVm.action('Delete');
+      leakedResources.delete(JSON.stringify({name: cloudInitVmName, namespace: testName, kind: 'vm'}));
+    }, VM_BOOTUP_TIMEOUT * 3);
```

Comment:
\+ here?

Reply:
Yeah, I liked it at first that it kinda reflects number of VMs TC works with, but since almost all cloning  cases inherently work with 2 VMs, it's just better to create a new constant for it.